notebooks/RAG_evaluation.ipynb

```diff
@@ -0,0 +1,979 @@
+{
```

Comment:
<p>I think we can make this section shorter. It is an advanced topic, and we can assume that the reader knows what RAG is, so we can shorten the intro, but it's a good idea to show the illustration to highlight the point that there are many moving parts that can affect the performance. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>@MKhalusova I will then simply remove the 'What is RAG' section, what do you think?</p>

Reply:
Sure, but I would leave the image - it shows the complexity and all the moving parts involved.